typescript/appconfig-hosted-configuration-lambda-extension/index.ts

```diff
@@ -0,0 +1,89 @@
+import * as cdk from '@aws-cdk/core';
+import { CfnApplication, CfnEnvironment, CfnConfigurationProfile, CfnHostedConfigurationVersion, CfnDeploymentStrategy, CfnDeployment } from '@aws-cdk/aws-appconfig';
```

Comment:
You should space long imports on separate lines (or import `*` and use an accessor

```suggestion
import { 
	CfnApplication,
	CfnEnvironment,
	CfnConfigurationProfile,
	CfnHostedConfigurationVersion,
	CfnDeploymentStrategy,
	CfnDeployment
} from '@aws-cdk/aws-appconfig';
```